elixir/apps/domain/lib/domain/auth/adapters/email.ex

```diff
@@ -64,7 +64,7 @@ defmodule Domain.Auth.Adapters.Email do
   end
 
   defp identity_create_state(%Provider{} = _provider) do
-    sign_in_token = Domain.Crypto.rand_string()
+    sign_in_token = Domain.Crypto.rand_number(5) <> Domain.Crypto.rand_string(16)
```

Comment:
Thinking about it twice this probably is not a great idea to reduce the randomity that much: what stops me from brute-forcing 5-digit numbers? We should also track attempts and delete tokens after certain threshold

Reply:
Agreed, it wouldn't hurt to increase the `rand_number()` call to something larger.  Also agree with tracking attempts and deleting tokens after a threshold.  It'd be even better if we could temporarily ban requests from the IP, but that may be better suited to do in a WAF.

Reply:
Pushed a fix.